doc/telestrat.sgml

```diff
@@ -16,35 +16,76 @@ An overview over the Telestrat (Telemon 2.4 & Telemon 3.x : http://orix.oric.org
+This Cardridge is a Sedoric ROM, Basic 1.1 ROM (Atmos), Basic 1.0 ROM (Oric-1).
+
+The main Telestrat's configuration is the Telemon/Hyperbasic Cardridge inserted 
+with Stratsed in floppy drive.
+
+Anyway, there is no way to load a tape file in Telemon/Hyperbasic mode without 
+alternative program.
+
+There is also no software to write a Stratsed dsk file on PC.
+
+This Telestrat target build an Orix binary file. But, in the future, it will be possible 
+to build a Stratsed disk. Orix uses the same systems calls than Telemon mode.
+
+That is why if you need to do software for telestrat target, you have the choice to :
+<itemize>
+<item>use cc65 Atmos target and start Telestrat in Atmos mode : a tape file is required
+<item>use cc65 Atmos target and start Telestrat in Stratoric mode : a dsk file or tape file is required
+<item>use cc65 Telestrat target and start Telestrat in Orix mode (see <url
+name="here" url="http://orix.oric.org/download/">)
+<item>use cc65 Telestrat target, remove Orix header from binary, code a dsk tool for Stratsed, 
+add Stratsed header on your binary, insert your binary on floppy disk (this solution will be possible is the future)
+</itemize>
+
+Telestrat (from cardridge) can handle 8 banks (from $C000 to $FFFF): Bank 0 is the overlay ram. Others banks can be ROM or RAM.
+
 <sect>Binary format<p>
 
 The standard binary output format generated the linker for the Telestrat
 target is a machine language program with a 20 bytes header described <url
-name="here" url="http://orix.oric.org/doku.php?id=orix:header">
+name="here" url="http://orix.oric.org/orix-header/">
 
 This header is used for Telemon 3.0.
 
-Anyway, for Telemon 2.4, there is no file management, there is no TAPE routine in Telemon, there is no way to load a binary easily.
+Anyway, for Telemon 2.4, there is no file management, there is no TAPE routine 
+in Telemon, there is no way to load a binary easily.
 
-Stratsed (the Telestrat operating system) handles files management. Stratsed is loaded to memory from floppy disk.
+Stratsed (the Telestrat operating system) handles files management. Stratsed 
+is loaded to memory from floppy disk. Stratsed vector are declared in asminc/telestrat.inc.
+But, reverse engineering is required to find how theses vectors works. Please, note that 
+Stratsed is located in overlay memory (bank 0)
 
 There is no tool to insert a binary in a Stratsed floppy disk.
 
 The only way to load a binary (for Telemon 2.4) is to:
 <itemize>
 <item>remove the 20 bytes header
 <item>download <url name="osdk" url="http://osdk.defence-force.org/index?page=download">
-<item>use Floppybuilder in OSDK to insert the binary with the tool (please read FloppyBuilder manual to insert your binary, and to start microdisc boot sector when Telestrat starts)
+<item>use Floppybuilder in OSDK to insert the binary with the tool (please read 
+FloppyBuilder manual to insert your binary, and to start microdisc boot sector 
```

Comment:
Fixed

Reply:
Hm, according to e.g. https://www.grammarly.com/blog/comma-before-and/ the comma seems wrong here.

Reply:
I misread it because some words are missing.  I now realize that Oliver is correct because it is actually a two-item list.
That comma should be removed.  And, the missing words should be added.

```html
<item>use Floppybuilder in OSDK to insert the binary with the tool (please read
the FloppyBuilder manual to learn how to insert your binary and how to start
the Microdisc boot sector when Telestrat starts)
```

Reply:
Fixed